Makefile

```diff
@@ -8,4 +8,11 @@ github_actions:
 	python3 -m venv ${VENV} && \
 		${VENV}/bin/python3 -m pip install --upgrade pip && \
 		${VENV}/bin/python3 -m pip install ./z-quantum-core && \
-		${VENV}/bin/python3 -m pip install -e '.[develop]'
\ No newline at end of file
+		${VENV}/bin/python3 -m pip install -e '.[develop]'
+
+build-system-deps:
+	$(PYTHON) -m pip install setuptools wheel "setuptools_scm>=6.0"
+
+
+build-system-deps:
```

Comment:
Looks like build-system-deps got duplicated

Reply:
Yup, fixed that!

Reply:
https://github.com/zapatacomputing/z-quantum-qcbm/pull/86#:~:text=deps%20to%20Makefile.-,2736de1,-max%2Dradin%20requested